doc/ledger3.texi

```diff
@@ -9677,9 +9677,10 @@ to the main body of the documentation.
 
 @menu
 * Cookbook::
+* Baseline Test Types::
 @end menu
 
-@node Cookbook,  , Miscellaneous Notes, Miscellaneous Notes
+@node Cookbook, Baseline Test Types, Miscellaneous Notes, Miscellaneous Notes
 @section Cookbook
```

Comment:
Without my commit, the ledger3.texi can not compile any more with makeinfo.
The point is that 114d61e is not yet pushed. I add to do it to test my changes.
Can you please as soon as you can this commit 114d61e?
Anyway, I closed this pull request, and split my pull request into several little commits.